assets/scss/01-atoms/_buttons.scss

```diff
@@ -14,40 +16,58 @@ button {
   @include ma-button("large");
 }
 
-//theme
-button {
-  font-family: "Texta", "Helvetica", "Arial", "sans-serif";
-  font-weight: 400;
-}
-
 .ma__button {
   @include ma-button("normal","solid",$c-primary);
 }
 
-.ma__button--minor {
-  @include ma-button("normal","outline",$c-primary);
+.ma__button--c-primary-alt {
+  @include ma-button("normal","solid",$c-primary-alt);
+}
+
+.ma__button--c-highlight {
+  @include ma-button("normal","solid",$c-highlight);

```

Comment:
this highlight button is going to fail accessibility

Reply:
for lighter colors the foreground color should use $c-black
![screen shot 2018-09-19 at 4 40 43 pm](https://user-images.githubusercontent.com/5789411/45780094-cb48e300-bc2a-11e8-80d3-5c6d7353753b.png)